python/turbine_models/tests/llama_test.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import turbine_models.custom_models.stateless_llama as llama
+import unittest
+import os
+
+
+class LLamaTest(unittest.TestCase):
+    def testExportTransformerModel(self):
```

Comment:
This is really only testing generating an f32 non quantized torch ir right now. Is there a plan to scale out for all permutations of vmfb/f16/quantized?

Reply:
Yes, just an initial setup

Reply:
makes sense

Reply:
took about 5 min currently. Maybe we can do just a few permutations and cover the flags that way

Reply:
5 mins per could balloon pretty quick but having some basic at least default settings would probably work well